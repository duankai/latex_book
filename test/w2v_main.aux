\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\providecommand*\tocstyle@set@width[4]{}
\tocstyle@set@width {unum}{toc}{}{11.74988pt}
\tocstyle@set@width {num}{toc}{0}{11.74988pt}
\tocstyle@set@width {skip}{toc}{0}{0.0pt}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}next}{5}{chapter.1}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
