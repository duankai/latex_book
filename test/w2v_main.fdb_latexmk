# Fdb version 3
["pdflatex"] 1593407172 "d:/latex_book/test/w2v_main.tex" "d:/latex_book/test/w2v_main.pdf" "w2v_main" 1593407173
  "d:/latex_book/test/w2v_main.aux" 1593407173 1014 4debac1b7a5f850439d1d29ddf88aa2e ""
  "d:/latex_book/test/w2v_main.tex" 1593401670 1276 dc37456d6cf9500657241d195cebff7d ""
  "f:/texlive/2020/texmf-dist/fonts/map/fontname/texfonts.map" 1591955098 3524 cb3e574dea2d1052e39280babc910dc8 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/arphic/gbsnu/gbsnu7b.tfm" 1591953924 1272 5dafd399d6bf482b8741eef5ef642512 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/arphic/gbsnu/gbsnu8b.tfm" 1591953924 1260 8850bb721613bfe532fa7baaee0e12b7 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/amsfonts/cmextra/cmex7.tfm" 1591953865 1004 54797486969f23fa377b128694d548df ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/amsfonts/cmextra/cmex8.tfm" 1591953865 988 bdf658c3bfc2d96d3c8b02cfc1c94c20 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/amsfonts/symbols/msam10.tfm" 1591953865 916 f87d7c45f9c908e672703b83b72241a3 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/amsfonts/symbols/msam7.tfm" 1591953865 928 2dc8d444221b7a635bb58038579b861a ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/amsfonts/symbols/msbm10.tfm" 1591953865 908 2921f8a10601f252058503cc6570e581 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/amsfonts/symbols/msbm7.tfm" 1591953865 940 228d6584342e91276bf566bcf9716b83 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmbx12.tfm" 1591954366 1324 c910af8c371558dc20f2d7822f66fe64 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmex10.tfm" 1591954366 992 662f679a0b3d2d53c1b94050fdaa3f50 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmmi12.tfm" 1591954366 1524 4414a8315f39513458b80dfc63bff03a ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmmi6.tfm" 1591954366 1512 f21f83efb36853c0b70002322c1ab3ad ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmmi8.tfm" 1591954366 1520 eccf95517727cb11801f4f1aee3a21b4 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmr12.tfm" 1591954366 1288 655e228510b4c2a1abe905c368440826 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmr17.tfm" 1591954366 1292 296a67155bdbfc32aa9c636f21e91433 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmr6.tfm" 1591954366 1300 b62933e007d01cfd073f79b963c01526 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmr8.tfm" 1591954366 1292 21c1c5bfeaebccffdb478fd231a0997d ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmsl12.tfm" 1591954366 1504 f280e6069e577ca4a981806c471fe65c ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmsy10.tfm" 1591954366 1124 6c73e740cf17375f03eec0ee63599741 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmsy6.tfm" 1591954366 1116 933a60c408fc0a863a92debe84b2d294 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmsy8.tfm" 1591954366 1120 8b7d695260f3cff42e636090a8002094 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmtt10.tfm" 1591954366 768 1321e9409b4137d6fb428ac9dc956269 ""
  "f:/texlive/2020/texmf-dist/fonts/tfm/public/cm/cmtt12.tfm" 1591954366 772 9a936b7f5e2ff0557fce0f62822f0bbf ""
  "f:/texlive/2020/texmf-dist/fonts/type1/arphic/gbsnu/gbsnu7b.pfb" 1591953925 62564 b4aebbf8bf205ceba50db7232b2e67a0 ""
  "f:/texlive/2020/texmf-dist/fonts/type1/arphic/gbsnu/gbsnu8b.pfb" 1591953925 48076 c8846bc6de91744dce393b7116a8c31b ""
  "f:/texlive/2020/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx12.pfb" 1591953865 32080 340ef9bf63678554ee606688e7b5339d ""
  "f:/texlive/2020/texmf-dist/fonts/type1/public/amsfonts/cm/cmr12.pfb" 1591953865 32722 d7379af29a190c3f453aba36302ff5a9 ""
  "f:/texlive/2020/texmf-dist/fonts/type1/public/amsfonts/cm/cmr17.pfb" 1591953865 32362 bc3f3eec7ab7d65fe700963d4017d32c ""
  "f:/texlive/2020/texmf-dist/fonts/type1/public/amsfonts/cm/cmsl12.pfb" 1591953865 34586 5a65dc15601f9c74fe3fb3f37b3b8b6f ""
  "f:/texlive/2020/texmf-dist/fonts/type1/public/amsfonts/cm/cmtt10.pfb" 1591953866 31099 342ef5a582aacbd3346f3cf4579679fa ""
  "f:/texlive/2020/texmf-dist/tex/context/base/mkii/supp-pdf.mkii" 1591956184 71627 94eb9990bed73c364d7f53f960cc8c5b ""
  "f:/texlive/2020/texmf-dist/tex/generic/atbegshi/atbegshi.sty" 1591953951 24708 5584a51a7101caf7e6bbf1fc27d8f7b1 ""
  "f:/texlive/2020/texmf-dist/tex/generic/bigintcalc/bigintcalc.sty" 1591954155 40635 c40361e206be584d448876bba8a64a3b ""
  "f:/texlive/2020/texmf-dist/tex/generic/bitset/bitset.sty" 1591954161 33961 6b5c75130e435b2bfdb9f480a09a39f9 ""
  "f:/texlive/2020/texmf-dist/tex/generic/etexcmds/etexcmds.sty" 1591954926 7734 b98cbb34c81f667027c1e3ebdbfce34b ""
  "f:/texlive/2020/texmf-dist/tex/generic/gettitlestring/gettitlestring.sty" 1591955183 8371 9d55b8bd010bc717624922fb3477d92e ""
  "f:/texlive/2020/texmf-dist/tex/generic/iftex/ifpdf.sty" 1591955407 480 5778104efadad304ced77548ca2184b1 ""
  "f:/texlive/2020/texmf-dist/tex/generic/iftex/iftex.sty" 1591955407 6501 4011d89d9621e0b0901138815ba5ff29 ""
  "f:/texlive/2020/texmf-dist/tex/generic/infwarerr/infwarerr.sty" 1591955428 8356 7bbb2c2373aa810be568c29e333da8ed ""
  "f:/texlive/2020/texmf-dist/tex/generic/intcalc/intcalc.sty" 1591955443 31769 002a487f55041f8e805cfbf6385ffd97 ""
  "f:/texlive/2020/texmf-dist/tex/generic/kvdefinekeys/kvdefinekeys.sty" 1591955599 5412 d5a2436094cd7be85769db90f29250a6 ""
  "f:/texlive/2020/texmf-dist/tex/generic/kvsetkeys/kvsetkeys.sty" 1591955601 13807 952b0226d4efca026f0e19dd266dcc22 ""
  "f:/texlive/2020/texmf-dist/tex/generic/ltxcmds/ltxcmds.sty" 1591955902 18552 1e1cc7b75da0dfaacce7cdcb27d306bf ""
  "f:/texlive/2020/texmf-dist/tex/generic/pdfescape/pdfescape.sty" 1591956551 19007 15924f7228aca6c6d184b115f4baa231 ""
  "f:/texlive/2020/texmf-dist/tex/generic/uniquecounter/uniquecounter.sty" 1591957947 7008 f92eaa0a3872ed622bbf538217cd2ab7 ""
  "f:/texlive/2020/texmf-dist/tex/latex/amsfonts/amsfonts.sty" 1591953866 5949 3f3fd50a8cc94c3d4cbf4fc66cd3df1c ""
  "f:/texlive/2020/texmf-dist/tex/latex/amsfonts/amssymb.sty" 1591953866 13829 94730e64147574077f8ecfea9bb69af4 ""
  "f:/texlive/2020/texmf-dist/tex/latex/amsfonts/umsa.fd" 1591953866 961 6518c6525a34feb5e8250ffa91731cff ""
  "f:/texlive/2020/texmf-dist/tex/latex/amsfonts/umsb.fd" 1591953866 961 d02606146ba5601b5645f987c92e6193 ""
  "f:/texlive/2020/texmf-dist/tex/latex/amsmath/amsbsy.sty" 1591953869 2211 ca7ce284ab93c8eecdc6029dc5ccbd73 ""
  "f:/texlive/2020/texmf-dist/tex/latex/amsmath/amsgen.sty" 1591953869 4161 7f6eb9092061a11f87d08ed13515b48d ""
  "f:/texlive/2020/texmf-dist/tex/latex/amsmath/amsmath.sty" 1591953869 85660 baee036978c7a91f4e2bba43f05e5945 ""
  "f:/texlive/2020/texmf-dist/tex/latex/amsmath/amsopn.sty" 1591953869 4116 32e6abd27229755a83a8b7f18e583890 ""
  "f:/texlive/2020/texmf-dist/tex/latex/amsmath/amstext.sty" 1591953869 2432 8ff93b1137020e8f21930562a874ae66 ""
  "f:/texlive/2020/texmf-dist/tex/latex/atveryend/atveryend.sty" 1591953954 19336 ce7ae9438967282886b3b036cfad1e4d ""
  "f:/texlive/2020/texmf-dist/tex/latex/auxhook/auxhook.sty" 1591953968 3935 57aa3c3e203a5c2effb4d2bd2efbc323 ""
  "f:/texlive/2020/texmf-dist/tex/latex/base/bk12.clo" 1591955629 8283 e1a5378f1428da595f40ba12daa0da23 ""
  "f:/texlive/2020/texmf-dist/tex/latex/base/book.cls" 1591955629 23147 87043b6c3ff70b5e5185aee55d462dd7 ""
  "f:/texlive/2020/texmf-dist/tex/latex/base/fontenc.sty" 1591955629 4947 0c2888dd88121ae675fc6e82213623ba ""
  "f:/texlive/2020/texmf-dist/tex/latex/base/inputenc.sty" 1591955629 5050 8933a39ad74377accd18991c5eb90c58 ""
  "f:/texlive/2020/texmf-dist/tex/latex/cjk/texinput/CJK.enc" 1591954343 29591 b3c404288523518719bdabea6811228d ""
  "f:/texlive/2020/texmf-dist/tex/latex/cjk/texinput/CJK.sty" 1591954343 32721 8e131bb8a9a2f3a0ef2f239af995b5af ""
  "f:/texlive/2020/texmf-dist/tex/latex/cjk/texinput/CJKutf8.sty" 1591954343 24621 8f4acdc36d36e5d58fc45253e3d4835b ""
  "f:/texlive/2020/texmf-dist/tex/latex/cjk/texinput/UTF8/UTF8.bdg" 1591954344 3857 3a3560ad093725b03513437085b094d2 ""
  "f:/texlive/2020/texmf-dist/tex/latex/cjk/texinput/UTF8/UTF8.chr" 1591954344 8601 3d4d7b821275de67f96f343619665314 ""
  "f:/texlive/2020/texmf-dist/tex/latex/cjk/texinput/UTF8/UTF8.enc" 1591954344 6111 2c4d70f06582f053f5e97e2be67b4899 ""
  "f:/texlive/2020/texmf-dist/tex/latex/cjk/texinput/UTF8/c70gbsn.fd" 1591954344 542 df90281be7edc0efb589d453e3f660f0 ""
  "f:/texlive/2020/texmf-dist/tex/latex/cjk/texinput/UTF8/c70gbsn.fdx" 1591954344 497 dc1fc2d256cc2db4d006c7be1f6f48f9 ""
  "f:/texlive/2020/texmf-dist/tex/latex/cjk/texinput/mule/MULEenc.sty" 1591954344 12177 49651d89af4c43f72d2608ca28d9934d ""
  "f:/texlive/2020/texmf-dist/tex/latex/colortbl/colortbl.sty" 1591954436 10793 d0af3aa11e27ae35ba4685b17597b122 ""
  "f:/texlive/2020/texmf-dist/tex/latex/enumitem/enumitem.sty" 1591954876 51697 f8f08183cd2080d9d18a41432d651dfb ""
  "f:/texlive/2020/texmf-dist/tex/latex/epstopdf-pkg/epstopdf-base.sty" 1591954890 13886 d1306dcf79a944f6988e688c1785f9ce ""
  "f:/texlive/2020/texmf-dist/tex/latex/graphics-cfg/color.cfg" 1591955255 1213 620bba36b25224fa9b7e1ccb4ecb76fd ""
  "f:/texlive/2020/texmf-dist/tex/latex/graphics-cfg/graphics.cfg" 1591955255 1224 978390e9c2234eab29404bc21b268d1e ""
  "f:/texlive/2020/texmf-dist/tex/latex/graphics-def/pdftex.def" 1591955255 17448 c8e20d5f7e69b0cbecf82ab29daeebd9 ""
  "f:/texlive/2020/texmf-dist/tex/latex/graphics/color.sty" 1591955253 7107 12f1c5ba3c6e4a1dde3cb5a3ae4b2cd1 ""
  "f:/texlive/2020/texmf-dist/tex/latex/graphics/dvipsnam.def" 1591955253 4944 1ddc361527c0bcf3d7b9f9b0c3c28113 ""
  "f:/texlive/2020/texmf-dist/tex/latex/graphics/graphics.sty" 1591955253 16932 04729abe63b66ec59ea56edcd722b058 ""
  "f:/texlive/2020/texmf-dist/tex/latex/graphics/graphicx.sty" 1591955253 9067 1b996612394a52e1efe89c8bfe8a5892 ""
  "f:/texlive/2020/texmf-dist/tex/latex/graphics/keyval.sty" 1591955253 2590 e3b24ff953e5b58d924f163d25380312 ""
  "f:/texlive/2020/texmf-dist/tex/latex/graphics/trig.sty" 1591955254 3976 d7fa7d81d2870d509d25b17d0245e735 ""
  "f:/texlive/2020/texmf-dist/tex/latex/hycolor/hycolor.sty" 1591955356 17914 4c28a13fc3d975e6e81c9bea1d697276 ""
  "f:/texlive/2020/texmf-dist/tex/latex/hyperref/hpdftex.def" 1591955361 50630 3d9728faf8630190cf601ce2cbe470d9 ""
  "f:/texlive/2020/texmf-dist/tex/latex/hyperref/hyperref.sty" 1591955361 238752 60dd338d71b6a4ab2192131f73dc908b ""
  "f:/texlive/2020/texmf-dist/tex/latex/hyperref/nameref.sty" 1591955361 13244 0070bcab7b5a88187847128d22faf4d8 ""
  "f:/texlive/2020/texmf-dist/tex/latex/hyperref/pd1enc.def" 1591955361 14134 32b36577d311ddb6522413c7581ee968 ""
  "f:/texlive/2020/texmf-dist/tex/latex/koma-script/tocstyle.sty" 1591955568 35357 befd57fb5a51f4df4885417f5bc60dce ""
  "f:/texlive/2020/texmf-dist/tex/latex/kvoptions/kvoptions.sty" 1591955600 22520 c4c2dab203104295e1e618be7e5c0f5b ""
  "f:/texlive/2020/texmf-dist/tex/latex/l3backend/l3backend-pdfmode.def" 1591955607 25398 19ae1e59d7f4a47fefb457d851f6f503 ""
  "f:/texlive/2020/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg" 1591955707 678 4792914a8f45be57bb98413425e4c7af ""
  "f:/texlive/2020/texmf-dist/tex/latex/letltxmacro/letltxmacro.sty" 1591955749 5766 13a9e8766c47f30327caf893ece86ac8 ""
  "f:/texlive/2020/texmf-dist/tex/latex/listings/listings.cfg" 1591955829 1830 e31effa752c61538383451ae21332364 ""
  "f:/texlive/2020/texmf-dist/tex/latex/listings/listings.sty" 1591955829 80964 64e57373f36316e4a09b517cbf1aba2e ""
  "f:/texlive/2020/texmf-dist/tex/latex/listings/lstlang1.sty" 1591955829 204271 bae5b2d457283e99567249c1990510be ""
  "f:/texlive/2020/texmf-dist/tex/latex/listings/lstmisc.sty" 1591955829 77022 ee25ce086f4a79d8cf73bac6f94c02a5 ""
  "f:/texlive/2020/texmf-dist/tex/latex/pdftexcmds/pdftexcmds.sty" 1591956568 19963 36fd8e818f9f0f32e2db8413d4970122 ""
  "f:/texlive/2020/texmf-dist/tex/latex/refcount/refcount.sty" 1591957100 9878 9e94e8fa600d95f9c7731bb21dfb67a4 ""
  "f:/texlive/2020/texmf-dist/tex/latex/rerunfilecheck/rerunfilecheck.sty" 1591957129 9715 b051d5b493d9fe5f4bc251462d039e5f ""
  "f:/texlive/2020/texmf-dist/tex/latex/subfigure/subfigure.cfg" 1591957497 2062 a0e7d66e09e508f51289a656aec06ed2 ""
  "f:/texlive/2020/texmf-dist/tex/latex/subfigure/subfigure.sty" 1591957497 15188 91281c7ddbccfa54a8e0c3b56ab5aa72 ""
  "f:/texlive/2020/texmf-dist/tex/latex/titlesec/titlesec.sty" 1591957770 48596 2b6a95da931c07a430b1a61904aaa42d ""
  "f:/texlive/2020/texmf-dist/tex/latex/tools/array.sty" 1591957812 12560 ce3f59ceae9d9a27bfe037d6bf1d903c ""
  "f:/texlive/2020/texmf-dist/tex/latex/tools/indentfirst.sty" 1591957812 1619 d4cd7231755dd1dfa89af8ceddfb0e4d ""
  "f:/texlive/2020/texmf-dist/tex/latex/url/url.sty" 1591957976 12796 8edb7d69a20b857904dd0ea757c14ec9 ""
  "f:/texlive/2020/texmf-dist/tex/latex/xcolor/svgnam.def" 1591958090 4620 ef4794ae13f75d0c199ba494bc41ff4b ""
  "f:/texlive/2020/texmf-dist/tex/latex/xcolor/xcolor.sty" 1591958090 55589 34128738f682d033422ca125f82e5d62 ""
  "f:/texlive/2020/texmf-dist/web2c/texmf.cnf" 1591955576 39451 60d7debd9ef8ece44cbe965493576c2d ""
  "f:/texlive/2020/texmf-var/fonts/map/pdftex/updmap/pdftex.map" 1591958267 4851253 cba6def0686b36ecfe098089d8ce9fad ""
  "f:/texlive/2020/texmf-var/web2c/pdftex/pdflatex.fmt" 1591958410 2432373 ab0b78950a85f54f8a50cc9ae5dfe188 ""
  "f:/texlive/2020/texmf.cnf" 1591958243 673 4ae7b2f49cee444c5343a45b5d0f169c ""
  "pythonhighlight.sty" 1593407168 5149 5735df97c76b0e369816541e42662aca ""
  "w2v_main.aux" 1593407173 1014 4debac1b7a5f850439d1d29ddf88aa2e "pdflatex"
  "w2v_main.out" 1593407173 39 52c107ac8ebf52ccae3b8da5c589cd65 "pdflatex"
  "w2v_main.tex" 1593401670 1276 dc37456d6cf9500657241d195cebff7d ""
  "w2v_main.toc" 1593407173 60 fb55ab29cfeb43e8a555eb36de9fb12d "pdflatex"
  (generated)
  "w2v_main.pdf"
  "w2v_main.toc"
  "w2v_main.aux"
  "d:/latex_book/test/w2v_main.pdf"
  "d:/latex_book/test/w2v_main.log"
  "w2v_main.out"
  "w2v_main.log"
