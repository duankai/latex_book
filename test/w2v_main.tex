\documentclass[a4paper,12pt]{book}
\usepackage{CJKutf8}
\usepackage{titlesec}
\usepackage{graphicx} 
\usepackage{subfigure}
\usepackage{amsbsy}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{array}
\usepackage[colorlinks,linkcolor=blue]{hyperref}
\usepackage{enumitem}
\usepackage{color}
\usepackage{graphicx}
\makeatletter
\color{black}
\let\default@color\current@color
\makeatother
\usepackage{pythonhighlight}
%\RequirePackage[tocindentauto,tocfullflat]{tocstyle}
% \usepackage[hyperref, UTF8]{ctex}
\newtheorem{definition}{Definition}


\begin{CJK*}{UTF8}{gbsn}
    %%*****************文本设置********************
    \RequirePackage{indentfirst}
    \setlength{\parindent}{2em}
    %% 行距
    \linespread{1.3}
    \selectfont
    % 页面顶行空白
    \setlength{\topskip}{5ex}
    % 段间距
    \setlength{\parskip}{1ex}
    %item 的缩进%
    \setlength{\itemindent}{2em}
    %%********************************************
    
    \begin{document}
    \title{Word2vec笔记}
    \author{Kevin}
    \date{2020-06-28}
    \maketitle

    \newpage
    \tableofcontents
    
    \chapter{next}

    \begin{python}
        def main():
            return
    \end{python}
\end{CJK*}
\end{document}
