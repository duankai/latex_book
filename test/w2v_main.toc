\contentsline {chapter}{\numberline {1}next}{5}{chapter.1}%
