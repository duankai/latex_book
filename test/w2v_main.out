\BOOKMARK [0][-]{chapter.1}{next}{}% 1
